### PR TITLE
Support byte array result when comparing

### PR DESCRIPTION
**Pull request by kabunchi**

### CODE REVIEW

File: benchto-driver/src/main/java/io/trino/benchto/driver/utils/QueryUtils.java [hunk 1]
```diff
@@ -130,7 +131,7 @@ private static String resultRowToString(ResultSet resultSet)
                 .map(i -> {
                     try {
                         Object value = resultSet.getObject(i);
-                        return value == null ? "" : value.toString();
+                        return value == null ? "" : (value instanceof byte[]) ? Arrays.toString((byte[]) value) : value.toString();
```
Comment (nineinchnick) [inline benchto-driver/src/main/java/io/trino/benchto/driver/utils/QueryUtils.java:134]: nit: nesting ternary operators doesn't look too good, how about splitting this up?
```suggestion
                        if (value == null) {
                             return "";
                        }
                        if (value instanceof byte[]) {
                             return Arrays.toString((byte[]) value);
                        }
                        return value.toString();
```

**Review by nineinchnick — APPROVED** [review]

**Review by przemekak — APPROVED** [review]

**Review by MiguelWeezardo — APPROVED** [review]

**Review by sopel39 — APPROVED** [review]
lgtm % @nineinchnick comment.

When would `byte[]` array be result of benchmark query?

**Comment by sopel39** [comment]
@kabunchi please ask for rereview when you address comments

**Comment by kabunchi** [comment]
@sopel39 we get `byte[]` as a result of `checksum(col)`
Also, fixed the comment, code ready for review.

**Comment by kabunchi** [comment]
FYI, also tested this branch on a private run of a branch in `benchmarks-gha` having `checksum` in its queries.

**Pull request merged**